src/components/FeaturedInsights/FeaturedInsightsHorizontal.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import InsightsFeatured from '../Insight/InsightsFeatured'
+import styles from './FeaturedInsightsHorizontal.module.scss'
+
+const FeaturedInsightsHorizontal = ({ maxLines, multilineTextId }) => (
+  <section className={styles.insights}>
+    <h4 className={styles.insights__title}>Featured insights</h4>
+    <div className={styles.insights__wrapper}>
+      <div className={styles.insights__scrollableWrapper}>
+        <div className={styles.insights__scrollable}>
+          <InsightsFeatured
+            maxLines={maxLines}
+            multilineTextId={multilineTextId}
```

Comment:
@haritonasty Do we really need this arg? What is it?

Reply:
those props for multilineText component. In InsightsFeatured component it can be different (on mobile three lines, on desktop - two. I pass it form parent, but yes, I think I should set values here, not in parent.

Reply:
@Partysun  no, I can't. I can do it only in FeaturedInsightsHorizontal, not in FIScrollable and FIWithTitle. They  can have different width on different pages, and multilineTextId depends on this.